modules/ocf_mysql/manifests/init.pp

```diff
@@ -1,30 +1,31 @@
 class ocf_mysql {
-  file {
-    # preseed root password
-    '/opt/share/puppet/mariadb-server-10.0.preseed':
-      mode      => '0600',
-      source    => 'puppet:///private/mariadb-server-10.0.preseed',
-      show_diff => false;
 
+  include ocf::ssl::default;
+
+  user { 'mysql':
+    groups => ['ssl-cert'];
+  }
```

Comment:
what creates the `mysql` user? the package install? if so, this should probably depend on the package, or it might fail on the first puppet run

Reply:
You probably want `system => true`